components/icons/fastapi.tsx

```diff
@@ -0,0 +1,23 @@
+export const FastApiIcon = () => {
+  return (
+    <div className="flex justify-center items-center  icon-integration">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        xmlnsXlink="http://www.w3.org/1999/xlink"
+        aria-hidden="true"
```

Comment:
maybe we should add this to all icons? I am not sure accessibility-wise what makes more sense